packages/forklift-console-plugin/src/modules/Plans/views/details/tabs/VirtualMachines/Migration/MigrationVirtualMachinesList.tsx

```diff
@@ -146,6 +147,7 @@ const fieldsMetadataFactory: ResourceFieldFactory = (t) => [
 ];
 
 const PageWithSelection = StandardPageWithSelection<VMData>;
+const PageWithExpansion = StandardPageWithExpansion<VMData>;
```

Comment:
If it's used only 1 time, do we need to declare a const?

Reply:
No not needed, I've actually never used/seen a component with a generic type before, but we can just call it like `<StandardPageWithSelection<VMData> {...extendedProps} />`